app/controllers/ems_common.rb

```diff
@@ -832,6 +871,40 @@ def get_form_vars
     set_verify_status
   end
 
+  def set_record_creds(ems)
+    creds = {}
+    creds[:default] = {
+      :userid   => @edit[:new][:default_userid],
```

Comment:
Can we also do something like this?

```ruby
edit_new = @edit[:new]
```

And replace all of the `@edit[:new]` calls... or even better, call `set_record_creds(ems, @edit[:new])` like that so then the method signature would be `def set_record_creds(ems, edit_new)`.  That would eliminate the knowledge of the ever present `@edit` and you can test this method without setting an instance variable.

Then, we could try figure out why we're building the `@edit[:new]` with a different structure than we need as the result of this method... for another time.